openvdb/openvdb/PlatformConfig.h

```diff
@@ -2,25 +2,7 @@
 // SPDX-License-Identifier: MPL-2.0
 ///
 /// @file PlatformConfig.h
+/// @note PlatformConfig.h will be marked as deprecated in the future
 
-#ifndef OPENVDB_PLATFORMCONFIG_HAS_BEEN_INCLUDED
-#define OPENVDB_PLATFORMCONFIG_HAS_BEEN_INCLUDED
+#include "Platform.h"
 
-// Windows specific configuration
-#ifdef _WIN32
-
-    // By default, assume we're building OpenVDB as a DLL if we're dynamically
-    // linking in the CRT, unless OPENVDB_STATICLIB is defined.
-    #if defined(_DLL) && !defined(OPENVDB_STATICLIB) && !defined(OPENVDB_DLL)
-        #define OPENVDB_DLL
-    #endif
-
-    // By default, assume that we're dynamically linking OpenEXR, unless
-    // OPENVDB_OPENEXR_STATICLIB is defined.
-    #if !defined(OPENVDB_OPENEXR_STATICLIB) && !defined(OPENEXR_DLL)
-        #define OPENEXR_DLL
-    #endif
```

Comment:
I've moved the `OPENVDB_DLL` logic into `Platform.h` but have completely **removed** the `OPENEXR_DLL` logic. The only case where this will change behaviour for downstream users is:

 - On Windows, where a **shared** build OpenEXR is being used as a dependency (which is being made optional in 8.1) **and** our CMake `Find` modules are **not** being used, `OPENEXR_DLL` will **not** be defined. This define needs to be provided by the OpenEXR installation or configured by the user.

Technically this should never have lived in VDB - it's a configuration option that needs to be provided by EXR (either through pkgconfig, cmake find/config modules, exr headers etc). If we are nervous about removing this I can re-introduce it in `Platform.h` and we can address it later.